LBHackney-IT/lbh-fss-portal-frontend
# Feature/admin add service

**Description**

Allow admin users to add a new service and associate it with an existing service  

**Test instructions**

- pull this branch
- navigate to **/services** and click 'Add service' 
- on the first page use the autocomplete field to search for existing services 
- progress through the form and submit a service (the service will not be added as using mock data)
- switch to a VCSO user in the /mock-api/account/GET.js file, and check the Organisation page containing the autocomplete field is not visible and that page number is correct given the different number of pages  

![image](https://user-images.githubusercontent.com/69238887/101542831-48430d00-399b-11eb-97e7-9318f851f67c.png)


**Ticket**

https://app.clickup.com/t/a6zwvf
